setup.sh

```diff
@@ -26,11 +26,8 @@ while [ "$1" != "" ]; do
       exit
       ;;
     -p | --prefix)
-      PREFIX=$VALUE
-      ;;
-    *)
-      echo "ERROR: unknown parameter \"$PARAM\""
-      usage
+      PREFIX=$VAChangeMe
```

Comment:
@federicavil  this appears to be temporary code?  I don't see where VAChangeMe is set.